src/app/pages/TopicPage/TopicPage.jsx

```diff
@@ -41,7 +43,10 @@ const TopicPage = ({ pageData }) => {
     .replace('{x}', activePage)
     .replace('{y}', pageCount);
 
-  const pageTitle = `${title}, ${translatedPage}`;
+  const metadataBaseTitle = seoTitle || title;
+  const seoPaginatedTitle = `${metadataBaseTitle}, ${translatedPage}`;
+  const metadataTitle = activePage >= 2 ? seoPaginatedTitle : metadataBaseTitle;
```

Comment:
Can you add a unit tests fort this scenario?

Reply:
Coolio, i've asserted this pagination case now